components/collapsible-panel/README.md

```diff
@@ -204,9 +203,7 @@ Use an inline collapsible panel to progressively disclose sections of a complex
 ## Summary Items [d2l-collapsible-panel-summary-item]
 An optional summary can help the user understand what’s inside the collapsible panel without having to expand it. This can be helpful if the user needs more than the heading to explain what’s inside.
 
-More than one d2l-collapsible-panel-summary-item can be added to the summary slot, and each will appear on its own line.
-
-<!-- docs: demo live -->
+<!-- docs: demo -->
```

Comment:
Currently, the properties for the `d2l-collapsible-panel-summary-item` component won't be shown in the documentation. Would it make sense to have a simple demo initially for it like the demo in `Optional Slots`? That would allow us to display the code snippet and properties of the component and then we could put the more complex demo somewhere else to show off the fully functional component.

Reply:
We do have this code snippet for summary items, which shows the maximum functionality of the summary item, which seems sufficient to me. But I'm definitely open to other thoughts here.

![image](https://user-images.githubusercontent.com/13937038/212353344-7a7fdb81-2260-4560-be68-0daaeb429a14.png)